javaparser-core-generators/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>javaparser-parent</artifactId>
         <groupId>com.github.javaparser</groupId>
-        <version>3.24.2-SNAPSHOT</version>
+        <version>3.22.2-SNAPSHOT</version>
```

Comment:
Why did you modify the java parser version?

Reply:
This was for having two versions on my local maven, in the final version of this PR every version is reverted to its original one.